test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -136,42 +160,112 @@ TEST_P(OverrideHostTest, HappyPath) {
   SetListenerAndRouteConfiguration(balancer_.get(),
                                    BuildListenerWithStatefulSessionFilter(),
                                    default_route_config_);
-  EdsResourceArgs args({{"locality0", CreateEndpointsForBackends()}});
-  balancer_->ads_service()->SetEdsResource(BuildEdsResource(args));
+  balancer_->ads_service()->SetEdsResource(BuildEdsResource(
+      EdsResourceArgs({{"locality0",
+                        {CreateEndpoint(0, HealthStatus::HEALTHY),
+                         CreateEndpoint(1, HealthStatus::UNKNOWN)}}})));
   WaitForAllBackends(DEBUG_LOCATION);
   // First call gets the cookie. RR policy picks the backend we will use.
```

Comment:
Please update this comment.  We're no longer just getting a cookie for a random backend; instead, we're getting a cookie for backend 0 specifically.